automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/workspaces/monitor/SubmissionDetailsPage.scala

```diff
@@ -74,20 +74,20 @@ class SubmissionDetailsPage(namespace: String, name: String, var submissionId: S
   }
 
   /**
-    * Wait for Submission to complete. 10 seconds polling.
+    * Wait for Submission to complete. 15 seconds polling.
     *
-    * @param timeOut: Time out. Default set 20.minutes
+    * @param timeOut: Time out. Default set 25.minutes
     */
-  def waitUntilSubmissionCompletes(timeOut: FiniteDuration = 20.minutes): Unit = {
+  def waitUntilSubmissionCompletes(timeOut: FiniteDuration = 25.minutes): Unit = {
     logger.info(s"Waiting for Workflow Submission to complete in $timeOut with 10 seconds polling interval")
     Thread.sleep(10000) // 10 seconds pause before checking
-    retry[Boolean](10.seconds, timeOut) ({
+    retry[Boolean](15.seconds, timeOut) ({
```

Comment:
high cpu and disk IO from parallel JVM and executions in a lot of Chrome. longer polling interval for "slowing down".